yarn.lock

```diff
@@ -9216,7 +9214,7 @@ __metadata:
   languageName: node
   linkType: hard
 
-"eslint-plugin-prettier@npm:^5.2.1, eslint-plugin-prettier@npm:^5.2.6":
+"eslint-plugin-prettier@npm:^5.2.1":
```

Comment:
surprised we still have a dep on it somewhere

Reply:
unfortunately,
```
❯ yarn why eslint-plugin-prettier
├─ eslint-plugin-github@npm:5.1.6
│  └─ eslint-plugin-prettier@npm:5.2.6 (via npm:^5.2.1)
│
└─ eslint-plugin-github@npm:5.1.6 [61e80]
   └─ eslint-plugin-prettier@npm:5.2.6 [1c633] (via npm:^5.2.1 [1c633])
```